src/utils/analytics.ts

```diff
@@ -0,0 +1,26 @@
+import { logDebug } from '.';
+import { PREVIEW_SERVICE_URL } from '../constants';
+import { checkAuthentication } from './auth';
+
+export const sendCliCommandAnalytics = async (prompt: 'import' | 'deploy' | 'preview' | 'auth') => {
```

Comment:
🤔 same question from the other pr: 
Grammatically, does it make more sense to call this a command everywhere?